src/sentry/auth/access.py

```diff
@@ -340,7 +371,13 @@ def from_member(
     # network hops and needed in a lot of places
     requires_sso, sso_is_valid = _sso_params(member)
 
-    team_list = frozenset(member.get_teams())
+    team_memberships = frozenset(
```

Comment:
would it make sense to add a method to member called `get_team_memberships` for this logic?

Reply:
👍 https://github.com/getsentry/sentry/pull/31816/commits/0150ceda965aa714a551d0ba46fb501de3783cf6